packages/react/src/oidc/FetchToken.tsx

```diff
@@ -47,7 +49,14 @@ export const withOidcFetch = (fetch: Fetch = null, configurationName = defaultCo
 export const useOidcFetch = (fetch: Fetch = null, configurationName = defaultConfigurationName) => {
   const previousFetch = fetch || window.fetch;
   const getOidc = VanillaOidc.get;
-  const getOidcWithConfigurationName = () => getOidc(configurationName);
-  const newFetch = fetchWithToken(previousFetch, getOidcWithConfigurationName);
-  return { fetch: newFetch };
+
+  const memoizedFetchCallback = useCallback(
+      (arg1, arg2) => {
+        const getOidcWithConfigurationName = () => getOidc(configurationName);
+        const newFetch = fetchWithToken(previousFetch, getOidcWithConfigurationName);
+        return newFetch(arg1, arg2);
```

Comment:
I'm afraid this is a breaking change, as it makes the previously-optional second parameter required. Upgrading to the version with this change breaks existing code:

```
TS2554: Expected 2 arguments, but got 1.
  > 32 |     fetch(url)
```

You can also see this in the compiled type definition `node_modules/@axa-fr/react-oidc/dist/FetchToken.d.ts`:

```ts
export declare const useOidcFetch: (fetch?: Fetch, configurationName?: string) => {
    fetch: (arg1: any, arg2: any) => Promise<Response>;
};
```

I think you'll need to do some type trickery to handle this correctly - maybe something like `(resource: any, options?: any) => ...`, although I regret the `any`s - but I don't have that fully tested and wanted to flag this as something you might want to consider reverting in the meantime.